ubi_manifest/worker/tasks/depsolve.py

```diff
@@ -208,6 +208,7 @@ def _save(data: dict[str, list[UbiUnit]]) -> None:
     for repo_id, units in data.items():
         items = []
         for unit in units:
+            item = {}
```

Comment:
This can theoretically append an empty dict to `items` list (which is not desirable), so maybe it would be safer to append only if `item` is non-empty, or add else branch with `continue` which will skip the append and go to next iteration immediately.